test/mounted_files_test.py

```diff
@@ -117,46 +87,13 @@ def test_mounted_files_package(supports_dir, env_mount_files, servicer, server_u
     }
 
 
-def test_mounted_files_package_with_automount(supports_dir, env_mount_files, servicer, server_url_env, token_env):
-    p = subprocess.run(
-        ["modal", "run", "pkg_a.package"],
-        cwd=supports_dir,
-        env={**os.environ, "MODAL_AUTOMOUNT": "1"},
-    )
-    assert p.returncode == 0
-    files = set(servicer.files_name2sha.keys()) - set(env_mount_files)
-    assert files == {
-        "/root/pkg_a/__init__.py",
-        "/root/pkg_a/a.py",
-        "/root/pkg_a/b/c.py",
-        "/root/pkg_a/b/e.py",
-        "/root/pkg_a/d.py",
-        "/root/pkg_a/package.py",
-        "/root/pkg_a/script.py",
-        "/root/pkg_a/serialized_fn.py",
-        "/root/pkg_b/__init__.py",
-        "/root/pkg_b/f.py",
-        "/root/pkg_b/g/h.py",
-    }
-
-
 @skip_windows("venvs behave differently on Windows.")
 def test_mounted_files_sys_prefix(servicer, supports_dir, venv_path, env_mount_files, server_url_env, token_env):
     # Run with venv activated, so it's on sys.prefix, and modal is dev-installed in the VM
-    subprocess.run(
-        [venv_path / "bin" / "modal", "run", script_path], cwd=supports_dir, env={**os.environ, "MODAL_AUTOMOUNT": "1"}
-    )
+    subprocess.run([venv_path / "bin" / "modal", "run", script_path], cwd=supports_dir, env={**os.environ})
     files = set(servicer.files_name2sha.keys()) - set(env_mount_files)
-    # Assert we include everything from `pkg_a` and `pkg_b` but not `pkg_c` or `modal`
-    assert files == {
-        "/root/a.py",
-        "/root/b/c.py",
-        "/root/b/e.py",
-        "/root/script.py",
-        "/root/pkg_b/__init__.py",
-        "/root/pkg_b/f.py",
-        "/root/pkg_b/g/h.py",
-    }
+    # No automount, so we only have the entrypoint file
+    assert files == {"/root/script.py"}
```

Comment:
such a lonely script - I'm not sure we need this test anymore - the whole `sys_prefix` thing is because of auto mount logic determining which modules should be included and not based on that